mos_tests/nfv/test_nfv_live_migration.py

```diff
@@ -0,0 +1,72 @@
+#    Copyright 2016 Mirantis, Inc.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+import logging
+
+import pytest
+
+from mos_tests.nfv.base import TestBaseNFV
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.mark.check_env_('is_vlan')
+class TestLiveMigration(TestBaseNFV):
+
+    @pytest.mark.check_env_('has_2_or_more_computes', 'is_ceph_enabled')
+    @pytest.mark.testrail_id('838327')
+    def test_lm_ceph_for_huge_pages(self, os_conn, networks, volume, keypair,
+                                    nfv_flavor, security_group, aggregate):
+        hosts = aggregate.hosts
```

Comment:
`hosts` may be `[]` or contains only one host. In suck cases your test will fail, but it should be skipped. You can add checks of hosts count to `aggregate` fixture